### PR TITLE
Implement centralized logging feature

### PR DESCRIPTION
**Pull request by zsotroav**
Implement a centralized and standardized method for logging and printing messages to the console. This feature uses the built-in python logging library.

Format: `[LEVEL] YYYY/MM/DD hh:mm:ss @ MODULE - MESSAGE`

Example: `[INFO] 2021/07/02 20:57:00 @ main - Logged in as Aelita-β#1976`

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -17,6 +17,15 @@ services:
...
       - SUGGESTIONS_CHANNEL=
       - PYTHONUNBUFFERED=1
+
+      # level of logging displayed in the console (20 means INFO and above):
+      # 10 DEBUG
+      # 20 INFO
+      # 30 WARNING
+      # 40 ERROR
+      # 50 CRITICAL
+      - LOGGING_LEVEL=
```
Comment (duncte123) [inline docker-compose.yml:27]: what would be the default for this

Comment (zsotroav) [reply docker-compose.yml:27]: If left empty, the code would probably error out due to the way the reading of env values is implemented.

I would recommend info level for almost all cases. Debug will communicate most discord API interactions, and there are a lot of API messages that you don't need when running prod or even feature tests.

Comment (duncte123) [reply docker-compose.yml:27]: If you default it to info I'll mark it as approved

File: .env.example [hunk 2]
```diff
@@ -9,6 +9,16 @@ BOTCMDS_512711766845947922=688775582049239188
...
 
+# level of logging displayed in the console (20 means INFO and above):
+# 10 DEBUG
+# 20 INFO
+# 30 WARNING
+# 40 ERROR
+# 50 CRITICAL
+LOGGING_LEVEL=
+# YES or NO
+LOG_FILE=
```
Comment (duncte123) [inline .env.example:20]:
```suggestion
LOGGING_LEVEL=20
# True or False
LOG_FILE=False
```

File: docker-compose.yml [hunk 3]
```diff
@@ -17,6 +17,17 @@ services:
...
+
+      # YES or NO
+      - LOG_FILE=
```
Comment (duncte123) [inline docker-compose.yml:30]:
```suggestion
      - LOGGING_LEVEL=20

      # True or False
      - LOG_FILE=False
```

File: utils/logger.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+LOG_FORMAT = '[%(levelname)s] %(asctime)s @ %(module)s - %(message)s'
+DATE_FORMAT = '%Y/%m/%d %H:%M:%S'
+formatter = logging.Formatter(LOG_FORMAT, datefmt=DATE_FORMAT)
+
+ch.setFormatter(formatter)
+
+logger.addHandler(ch)
+
+# File handler
+if os.getenv('LOG_FILE') == 'YES':
```
Comment (duncte123) [inline utils/logger.py:31]:
```suggestion
if os.getenv('LOG_FILE') == 'True':
```

**Review by duncte123 — CHANGES_REQUESTED** [review]

**Review by duncte123 — APPROVED** [review]

**Pull request merged**